keyvalue/clients/src/main/java/tachyon/client/keyvalue/hadoop/KeyValueInputFormat.java

```diff
@@ -67,8 +67,8 @@
     List<InputSplit> splits = Lists.newArrayList();
     try {
       for (Path path : paths) {
-        List<PartitionInfo> partitionInfos =
-            mKeyValueMasterClient.getPartitionInfo(new TachyonURI(path.toString()));
+        List<PartitionInfo> partitionInfos = mKeyValueMasterClient.getPartitionInfo(new TachyonURI(
+            path.toString()));
```

Comment:
Good to know that's not strict, I thought we would write as long as possible before starting another line..